miro_preprocessor/miro_inventory/main.tf

```diff
@@ -0,0 +1,60 @@
+module "miro_inventory_lambda" {
+  source = "git::https://github.com/wellcometrust/terraform.git//lambda?ref=v1.0.0"
```

Comment:
nit: cleaner just to do `github.com/wellcometrust`, you don't need the `git::https://` prefix

Reply:
I _think_ this is a necessity when using the ref param (at least the terraform docs indicate so).